sql-plugin/src/main/scala/org/apache/spark/rapids/hybrid/HybridExecutionUtils.scala

```diff
@@ -98,6 +98,22 @@ object HybridExecutionUtils extends PredicateHelper {
         case _: BinaryType =>
           logWarning(s"Fallback to GpuScan because BinaryType is not supported: $field")
           true
+        // TODO: support CalendarIntervalType
+        case _: CalendarIntervalType =>
```

Comment:
Is it possible to add these new defined unsupported types as cases of `test_hybrid_parquet_read_fallback_to_gpu` ?

Reply:
Added case for `DayTimeInterval`.
But we don't have datagens for CalendarIntervalType, YearMonthIntervalType and UDT. So not straightforward to test them. I'd prefer to skip these tests if it's not so important, or spilt this out of this pr otherwise. Any suggestions?

Reply:
Sounds good to me